packages/react-card/Spec.md

```diff
@@ -102,21 +102,20 @@ Card goes for a more structural and generic approach to a card component and is
-| orientation | `vertical`, `horizontal`                                                             | `vertical` | Orientation of the card                                                                         |
-| size        | `smallest`, `smaller`, `small`, `medium`, `large`                                    | `medium`   | Define the minimum size of the card. Smaller sizes only apply to horizontal card                |
-| scale       | `fixed`, `auto-width`, `auto-height`, `auto`, `fluid-width`, `fluid-height`, `fluid` | `auto`     | Manages how the card handles it's scaling depending on the content                              |
-| appearance  | `filled`, `filled-alternative`, `outline`, `subtle`                                  | `filled`   | Define the appearance of the card                                                               |
-| selectable  | boolean                                                                              | false      | Makes the card selectable by adding a checkbox to the _Actions_ area                            |
-| selected    | boolean                                                                              | false      | Set to `true` if card is selected                                                               |
-| expandable  | boolean                                                                              | false      | Allow card to expand to show whole content                                                      |
-| disabled    | boolean                                                                              | false      | Makes the card and card selection disabled (not propagated to children)                         |
-| focusable   | boolean \| 'noTab' \| 'tabExit' \| 'tabOnly'                                         | false      | Sets the focus behavior for the card. If `true`, the card will have the 'noTab' focus behavior. |
+| Property    | Values                                                                      | Default    | Purpose                                                                                         |
+| ----------- | --------------------------------------------------------------------------- | ---------- | ----------------------------------------------------------------------------------------------- |
+| orientation | `vertical`, `horizontal`                                                    | `vertical` | Orientation of the card                                                                         |
+| size        | `smallest`, `smaller`, `small`, `medium`, `large`                           | `medium`   | Define the minimum size of the card. Smaller sizes only apply to horizontal card                |
+| scale       | `auto-width`, `auto-height`, `auto`, `fluid-width`, `fluid-height`, `fluid` | undefined  | Manages how the card handles it's scaling depending on the content                              |
```

Comment:
Dropped "fixed" prop as it does not add any styles to a Card component and is just the default state of it.

Reply:
`undefined` is default ? is this something that we want ?

Reply:
> to me, if there's a value to a prop, there's an outcome

I guess I don't follow? orientation uses `vertical` as default. with that what you're saying is invalid as I don;t have to specify `orientation` and it will apply a value.

In general I'd say the terminology is confusing here. I'd expect `scale` to have a default value `auto`, that's what I was referring ( that's the value you had there before )

Reply:
I agree with you, the only problem here is that `auto` has no styles. So what's the point of having a value that does nothing? If the default does nothing, why not keep it nullable? My reasoning was that it's clear that the default will have no impact. If it's not the case, I'll add an `auto` value again that does nothing.
To me, allowing a user to do `<Card scale="auto">` and the result be a div with no specific class seems misleading.